.github/workflows/build_project-awscdk-typescript.yml

```diff
@@ -28,4 +28,5 @@ jobs:
         with:
           node-version: 16
       - run: npm i -g aws-cdk
+      - run: npm i
```

Comment:
It's included in next step `npm ci-build` already

Reply:
```suggestion
```

Reply:
```
npm ERR! `npm ci` can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
```
this is the exact error. delete `package-lock.json` locally etc. to sort?

Reply:
that worked. reverted this line.

Reply:
Yep annoyance about node - get rid of package lock as well as node_modules and re-run npm i locally should solve it